config/config.go

```diff
@@ -404,6 +407,7 @@ type Experimental struct {
 	AuditLog              AuditLog              `yaml:"auditLog"`
 	AwsEnvironment        AwsEnvironment        `yaml:"awsEnvironment"`
 	AwsNodeLabels         AwsNodeLabels         `yaml:"awsNodeLabels"`
+	ClusterAutoScaler     ClusterAutoScaler     `yaml:"clusterAutoScaler"`
```

Comment:
nit: I prefer `ClusterAutoscaler` and `clusterAutoscaler` because the original CA is spelled `cluster-autoscaler`